### ISSUE TITLE
scp cannot copy multiple files at once as it claims to be capable of

### ISSUE THREAD

**Issue by svgPhoenix** [bitburner-official/bitburner-src#571]
help scp contradicts itself: 
The second argument passed in must be the hostname or IP of the target server. Examples:
 
    scp foo.script n00dles
 
    scp foo.script bar.script n00dles

I think the better solution would be to add the functionality implied by the second example, but technically just deleting it would fix the contradiction.

**Comment by MaWo2** [comment]
Maybe related to this: Using scp in a script to transfer several files and folders fails after the first folder.

sample script
(ServersOfTheWorld.txt contains a list of all servers)

```js
export async function main(ns) {
	var target = ns.read("ServersOfTheWorld.txt").split(",");
	var foldersToCopy = ["/simplehack", "/batchhack", "/loophack"];
	var filesToCopy = ["monitor.js", "n-monitor.js", "purchaseServer8GB.js", "connect-to.js", "shareme.js"];
	for (let i = 0; i < target.length; i++) {
		//copy files to servers
		for (let j = 0; j < foldersToCopy.length; j++) {
			await ns.scp(ns.ls("home", foldersToCopy[j]), target[i], "home");
		}
		await ns.scp(filesToCopy, target[i], "home");
	}	
}
```

Expected result:
- individual files (`filesToCopy`) and
- all files in all `foldersToCopy` are transfered to the servers

Actual result:
- all individual files are copied
- only the files in the first folder `/simplehack` are copied

**Comment by MaWo2** [comment]
If the leading slashes are removed from the folder names, it's working.

However, this not consistent with the tooltip-documentation of ns.scp, which claims that files in subfolders need the leading slash.

**Comment by Snarling** [comment]
Both of the issues here (terminal scp command not handling multiple files and ns.ls not matching when a leading slash is provided) should be fixed by #574 when it is merged

**Comment by Snarling** [comment]
Fixed by #574
